### PR TITLE
feat: overwrite prev/next link

### PR DESCRIPTION
**Pull request by Spice-Z**
This is built on top of https://github.com/vuejs/vitepress/pull/56, should be merged after it.
Enable to explicitly overwrite or disable prev/next link for individual pages setting like this
```
---
prev: false
next: /guide/getting-started
---
```

### CODE REVIEW

**Comment by kiaking** [comment]
@Spice-Z Hi! Could you rebase this one to the latest since #56 is merged now 🙏

**Comment by Spice-Z** [comment]
@kiaking
I merged latest change to this one!

File: src/client/theme-default/components/NextAndPrevLinks.ts [hunk 1]
```diff
@@ -1,13 +1,43 @@
 import { defineComponent, computed } from 'vue'
 import { usePageData } from 'vitepress'
+import { useSiteData } from 'vitepress'
```
Comment (kiaking) [inline src/client/theme-default/components/NextAndPrevLinks.ts:3]: Can we merge this one with the above import?

```suggestion
import { usePageData, useSiteData } from 'vitepress'
```

**Review by kiaking — CHANGES_REQUESTED** [review]
Thanks for the rebase! Just added 1 comment, please take a look.

File: src/client/theme-default/components/NextAndPrevLinks.ts (hunk 1 above)
Comment (Spice-Z) [reply src/client/theme-default/components/NextAndPrevLinks.ts]: Thank you!
Merged!!

**Review by kiaking — APPROVED** [review]
Nice! Thanks you so much for this 🙌

**Pull request merged**